nemo/core/classes/modelPT.py

```diff
@@ -80,27 +86,37 @@ def __init__(self, cfg: DictConfig, trainer: Trainer = None):
         self._scheduler = None
         self._trainer = trainer
```

Comment:
Related to `set_trainer()`, it seems like you just set it here.

Reply:
Since `_trainer` is a private method, which hopefully users should not see, I've added the public set_trainer method for restoration + fine tuning / continued training / training from scratch.